automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceSummaryPage.scala

```diff
@@ -186,8 +186,16 @@ class CloneWorkspaceModal(implicit webDriver: WebDriver) extends FireCloudView {
       textField(workspaceNameInput).value = name
     }
 
-    def readPresetAuthDomain(): Option[String] = {
-      find(presetAuthDomain).map(_.text)
+    def readAuthDomainGroups(): List[(String, Boolean)] = {
+      await visible authDomainGroupsQuery
+
+      findAll(authDomainGroupsQuery).map { element =>
+        (element.attribute("value").get, element.isEnabled)
+      }.toList
+    }
+
+    def readLockedAuthDomainGroups(): List[String] = {
+      readAuthDomainGroups().filterNot{ case (_, isEnabled) => isEnabled }.map { case (name, _) => name }
```

Comment:
I disagree somewhat... not necessarily with using a separate data-test-id but that it would be more robust. Regardless of the data-test-id, tests need to verify that the groups are locked and cannot be changed; there's no guarantee that they are actually locked just be cause the data-test-id implies it. At that point it doesn't really matter what the data-test-id is as long as it selects the right element(s).